### ISSUE TITLE
[Footer]: QA: Screen reader does not identify the clear button

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#8858]
### Description

Page Element:
Clear button in Choose a language dropdown

Issue:
Screen reader does not identify the clear button that is present after the dropdown

Expected:
Screen reader is expected to identify the "clear button" and the same should be keyboard operable for the SR users

User Impact:
When the screen reader doesn’t identify the page elements, it impacts the experience of users who are blind or have vision impairment. They can miss important information or fail to perform an action on the web page. 

### Component(s) impacted

Footer (Default)

### Browser

Firefox

### Carbon for IBM.com version

v1.34.0-rc.0

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://www.ibm.com/standards/carbon/web-components/?path=/story/

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by emyarod** [comment]
just to clarify, is the issue around the clear button for the search field here?

<img width="1440" alt="Screen Shot 2022-09-06 at 13 43 24" src="https://user-images.githubusercontent.com/8265238/188714352-d6ccff86-e1b6-4c48-8a8e-e16d1a44b83b.png">

**Comment by Rasubra8** [comment]
@emyarod 
Sorry. The component variant name is mentioned wrongly. It is “Footer language only” Please refer the screen short attached 
![Web component Footer language only](https://user-images.githubusercontent.com/28350461/191935336-b76fe89d-f9f7-41cd-9a49-a75fb8da810d.png)
here.

**Comment by emyarod** [comment]
@Rasubra8 I have been seeing this output in the screenreader. can you confirm if the issue is still present?

![image](https://user-images.githubusercontent.com/8265238/196252767-65f00b40-dd1d-4561-940b-4fb0c782bc31.png)

**Comment by Rasubra8** [comment]
@emyarod 
It looks like you are checking in Mac. The issue occurs in windows FF with JAWS.
Step to reproduce in Windows with jaws.
1. Turn on the JAWS screen reader in windows. 
2. Launch the page in FF
3. Use tab / shift tab and up/down arrow keys to navigate through the page elements.
JAWS does not identify the clear button
